docs/csharp/language-reference/toc.yml

```diff
@@ -629,3 +631,5 @@
     href: ../../../_csharplang/proposals/csharp-8.0/using.md
   - name: null coalescing assignment
     href: ../../../_csharplang/proposals/csharp-8.0/null-coalescing-assignment.md
+  - name: readonly instance members
```

Comment:
There's no pattern for heading style in this section. Some use lowercase, some use sentence case.